var/spack/repos/builtin/packages/py-dp-gp-cluster/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from os import unlink
+from glob import glob
+
+
+class PyDpGpCluster(PythonPackage):
+    """DP_GP_cluster clusters genes by expression over a time course using a
+       Dirichlet process Gaussian process model."""
+
+    homepage = "https://github.com/PrincetonUniversity/DP_GP_cluster"
+    git      = "https://github.com/PrincetonUniversity/DP_GP_cluster.git"
+
+    version('2019-09-22', commit='eec12e74219f916aa86e253783905f7b5e30f6f4')
+
+    depends_on('python@2.7:2.8', type=('build', 'run'))
+
+    depends_on('py-cython', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-cython', type='build')
```

Reply:
This package tries to load the cython module at runtime but it does seem to be optional, removed.